oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/LocalVocabularySelectField/LocalVocabularySelectField.jsx

```diff
@@ -10,22 +9,35 @@ import { i18next } from "@translations/oarepo_vocabularies_ui/i18next";
 export const serializedVocabularyItems = (vocabularyItems) =>
   vocabularyItems.map((vocabularyItem) => {
     const {
-      hierarchy: { title },
+      hierarchy: { title: titlesArray },
       text,
     } = vocabularyItem;
     const sections = [
-      ...title.map((title, index) => ({
-        content: title,
-        key: index,
-      })),
+      ...titlesArray.map((title, index) => {
+        if (index === 0) {
+          return {
+            content: <span>{title}</span>,
+            key: index,
```

Comment:
let's use `crypto.randomUUID()` as title key, `key: index` can be buggy between renders as it is not very consistent when items start (dis)appearing